doc/Command-Reference.md

```diff
@@ -6564,6 +6568,301 @@ This command is used to set the counter polling interval. Default is 20 seconds.
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#sflow)
 
+## SNMP
+
+### SNMP Show commands
+
+**show run snmp**
```

Comment:
Suggest using full, non-abbreviated command and group names in this reference guide (e.g., `show runningconfiguration snmp`. Please update throughout.

Reply:
Fixed all the show run snmp to be show runningconfiguration snmp.